src/dataprotection/azext_dataprotection/aaz/latest/dataprotection/backup_vault/_create.py

```diff
@@ -0,0 +1,491 @@
+            return cls._args_schema
+        cls._args_schema = super()._build_arguments_schema(*args, **kwargs)
+
+        # define Arg Group ""
+
+        _args_schema = cls._args_schema
+        _args_schema.resource_group = AAZResourceGroupNameArg(
+            required=True,
+        )
+        _args_schema.vault_name = AAZStrArg(
+            options=["--vault-name"],
+            help="The name of the backup vault.",
+            required=True,
+        )
+
+        # define Arg Group "Identity"
+
+        _args_schema = cls._args_schema
+        _args_schema.type = AAZStrArg(
+            options=["--type"],
+            arg_group="Identity",
+            help="The identityType which can be either SystemAssigned or None",
+        )
+
+        # define Arg Group "Monitoring Settings Azure Monitor Alert Settings"
+
+        _args_schema = cls._args_schema
+        _args_schema.azure_monitor_alerts_for_job_failures = AAZStrArg(
+            options=["--job-failure-alerts", "--azure-monitor-alerts-for-job-failures"],
+            arg_group="Monitoring Settings Azure Monitor Alert Settings",
+            help="Property that specifies whether built-in Azure Monitor alerts should be fired for all failed jobs.",
+            enum={"Disabled": "Disabled", "Enabled": "Enabled"},
+        )
+
+        # define Arg Group "Parameters"
+
+        _args_schema = cls._args_schema
+        _args_schema.e_tag = AAZStrArg(
+            options=["--e-tag"],
+            arg_group="Parameters",
+            help="Optional ETag.",
+        )
+        _args_schema.location = AAZResourceLocationArg(
+            arg_group="Parameters",
+            help="Location. Values from: `az account list-locations`. You can configure the default location using `az configure --defaults location=<location>`.",
+            required=True,
+            fmt=AAZResourceLocationArgFormat(
+                resource_group_arg="resource_group",
+            ),
+        )
+        _args_schema.tags = AAZDictArg(
+            options=["--tags"],
+            arg_group="Parameters",
+            help="Space-separated tags: key[=value] [key[=value] ...]. Use \"\" to clear existing tags.",
+        )
+
+        tags = cls._args_schema.tags
+        tags.Element = AAZStrArg()
+
+        # define Arg Group "Properties"
+
+        _args_schema = cls._args_schema
+        _args_schema.storage_setting = AAZListArg(
+            options=["--storage-setting"],
+            singular_options=["--storage-settings"],
+            arg_group="Properties",
+            help={"short-summary": "Storage Settings. Usage: --storage-setting \"[{type:'LocallyRedundant',datastore-type:'VaultStore'}]\"", "long-summary": "Multiple actions can be specified by using more than one --storage-setting argument.\nThe --storage-settings parameter exists for backwards compatibility. The updated command is --storage-setting.\nUsage for --storage-settings: --storage-settings  type=XX datastore-type=XX."},
```

Comment:
> Help-Entry: `dataprotection backup-vault create` - The following parameter help names are invalid: --storage-settings

Reply:
@kairu-ms May I ask if the `singular_options` --storage-settings cannot appear in the `long-summary` now?

Reply:
The issue was that the prior autorest-generated help text had `--storage-settings` in its help text. Solution was to remove that.